plugins/modules/azure_rm_aduser.py

```diff
@@ -119,6 +119,17 @@
             - The maximum length is 64 characters.Returned only on $select.
             - Supports $filter (eq, ne, not, ge, le, in, startsWith, and eq on null values).
         type: str
+    on_premises_extension_attributes:
+        description:
+            - Contains extensionAttributes1-15 for the user.
+            - These extension attributes are also known as Exchange custom attributes 1-15.
+            - For an onPremisesSyncEnabled user, the source of authority for this set of properties is the on-premises and is read-only.
+            - For a cloud-only user (where onPremisesSyncEnabled is false), these properties can be set during the creation or update of a user object.
+            - For a cloud-only user previously synced from on-premises Active Directory, these properties are read-only in Microsoft Graph but can be fully managed through the Exchange Admin Center or the Exchange Online V2 module in PowerShell.
```

Comment:
line too long(lines 128, 229, 322, 324, 326, 333, 358, 439)
```suggestion
            - For a cloud-only user previously synced from on-premises Active Directory, these properties are read-only in Microsoft Graph\
              but can be fully managed through the Exchange Admin Center or the Exchange Online V2 module in PowerShell.
```